packages/neo3-invoker/README.MD

```diff
@@ -10,9 +10,211 @@
 </p>
 
 # Neo3-Invoker
+
 Neo3-Invoker is a specification of how SmartContract client SDKs can interact with different invoker libraries such Neon-JS or WalletConnect.
 
 With Neo3-Invoker a SDK don't need to reimplement the wheel and can focus on the SmartContract communication format.
 Taking advantage of the declarative nature of the specification, the SDK can be used with different invoker libraries, it only depends on what the dApp developer wants to use.
 
 Visit the [main page](../../README.md) of the project.
+
+## Arguments examples:
+
+For these examples, we are using the [NeonInvoker](../neon-invoker/README.md) that is an implementation of Neo3Invoker, so we are starting from the point that NeonInvoker is already initialized.
+
+### Any
+
+It is expecting receive a HEX string as value.
+
+```ts
+const hexValue = 'HEX string'
+invoker.testInvoke({
+  invocations: [
+    {
+      operation: 'method',
+      scriptHash: '0x0000000000000000000000000000000000000000',
```

Comment:
this long scriptHash is a little bit distracting but I don't know if there is a way to improve this, let me know if you have any idea.

Reply:
I was using 0x0. Lucas recommended me to use this way for better understanding

Reply:
maybe 
```json
invocations: [
    {
      operation: 'method',
      scriptHash: contractSH,
```

Reply:
I've commited a change related to this topic: https://github.com/CityOfZion/neo3-invoker/pull/6/commits/8934df8ce268156f8ce153bb1254f28e943971f1